packages/svelte/tests/runtime-runes/samples/effect-tracking-unowned/main.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+	import { writable, fromStore, toStore } from "svelte/store";
+
+	const store = writable("previous");
+	let text = $derived(fromStore(store).current + " message");
+
+	const textStore = toStore(() => text);
```

Comment:
We can simplify this further

```suggestion
	text; // read derived in a non-tracking context
```